subnet/registry.go

```diff
@@ -131,13 +132,19 @@ func (esr *etcdSubnetRegistry) deleteSubnet(ctx context.Context, network, sn str
 	return esr.client().Delete(ctx, key, nil)
 }
 
-func (esr *etcdSubnetRegistry) watchSubnets(ctx context.Context, network string, since uint64) (*etcd.Response, error) {
-	key := path.Join(esr.etcdCfg.Prefix, network, "subnets")
+func (esr *etcdSubnetRegistry) watch(ctx context.Context, subpath string, since uint64) (*etcd.Response, error) {
+	key := path.Join(esr.etcdCfg.Prefix, subpath)
 	opts := &etcd.WatcherOptions{
 		AfterIndex: since,
 		Recursive:  true,
 	}
-	return esr.client().Watcher(key, opts).Next(ctx)
+	e, err := esr.client().Watcher(key, opts).Next(ctx)
+	return e, err
+}
+
+func (esr *etcdSubnetRegistry) getNetworks(ctx context.Context) (*etcd.Response, error) {
+	key := path.Join(esr.etcdCfg.Prefix, "network")
```

Comment:
Yeah, just found that out when refactoring the network/ code to watch networks.  It's bogus; removed.